specs/Standards/Sould Bound Token/README.md

```diff
@@ -0,0 +1,235 @@
+
+Two safeguards against misuse of recovery are contemplated. 1) Users cannot recover an SBT by themselves. The issuer, a DAO or a smart contract (eg: multisig) dedicated to manage the recovery should be assigned. 2) Whenever a recovery is triggered then the wallet from which the NFT was recovered gets blacklisted (indicator that the account identity is burned). Recovering one SBT triggers a blacklist that should apply for SBTs that share the same address.
+
+Soulbound tokens with expire date SHOULD have an option to be renewable. Examples include mandatory renewal with some frequency to check that the owner is still alive, or renew membership to a DAO that uses SBTs as membership gating.
+
+### Smart contract interface
+
+The Soulbound Token interface is a subset of NFT, hence the interface follows the [NEP-171](https://github.com/near/NEPs/blob/master/neps/nep-0171.md).
+
+```rust
+/// TokenMetadata defines attributes for each SBT token.
+pub struct TokenMetadata {
+    pub title: Option<String>, // ex. "fist bump with Robert"
+    pub description: Option<String>, // free-form description
+    pub media: Option<String>, // URL to associated media, preferably to decentralized, content-addressed storage
+    pub media_hash: Option<Base64VecU8>, // Base64-encoded sha256 hash of content referenced by the `media` field. Required if `media` is included.
+    pub copies: Option<u64>, // number of copies of this set of metadata in existence when token was minted.
+    pub issued_at: Option<u64>, // When token was issued or minted, Unix epoch in milliseconds
+    pub expires_at: Option<u64>, // When token expires, Unix epoch in milliseconds
+    pub starts_at: Option<u64>, // When token starts being valid, Unix epoch in milliseconds
+    pub updated_at: Option<u64>, // When token was last updated, Unix epoch in milliseconds
+    pub extra: Option<String>, // anything extra the SBT wants to store on-chain. Can be stringified JSON.
+    pub reference: Option<String>, // URL to an off-chain JSON file with more info.
+    pub reference_hash: Option<Base64VecU8>, // Base64-encoded sha256 hash of JSON from reference field. Required if `reference` is included.
+}
+
+
+trait SBT {
+    /**********
+    * QUERIES
+    **********/
+
+    /// get the information about specific token ID
+    fn sbt(&self, token_id: TokenId) -> Option<Token>;
+
+    /// returns total amount of tokens minted by this contract
+    fn sbt_total_supply(&self) -> U64;
+
+    /// returns total supply of SBTs for a given owner
+    fn sbt_supply_for_owner(&self, account: AccountId);
+
+    /// Query for sbt tokens
+    fn sbt_tokens(&self, from_index: Option<U64>, limit: Option<u32>) -> Vec<Token>;
+
+    /// Query sbt tokens by owner
+    fn sbt_tokens_for_owner(
+        &self,
+        account: AccountId,
+        from_index: Option<U64>,
+        limit: Option<u32>,
+    ) -> Vec<Token>;
```

Comment:
By the way, `Token` type is missing. Note that it seems that NEP-171 underspecifies `Token` type to only return `owner_id` and `token_id` while it returns metadata in the near-sdk-rs implementation which is pretty reasonable.